react/features/base/conference/actions.js

```diff
@@ -69,6 +72,16 @@ function _addConferenceListeners(conference, dispatch) {
     conference.on(
         JitsiConferenceEvents.STARTED_MUTED,
         () => {
+            const audioMuted = Boolean(conference.startAudioMuted);
+            const videoMuted = Boolean(conference.startVideoMuted);
+
+            sendEvent(
+                `startedmuted.audio.${audioMuted ? 'enabled' : 'disabled'}`);
```

Comment:
We also use the term of enabled and disabled audio or video, so wouldn't it be better to call those "on" and "off"?